src/js/themes/base.js

```diff
@@ -62,6 +62,10 @@ const colors = {
     light: 'brand',
   },
   focus: focusColor,
+  // "graph-0": 'accent-1',
+  // "graph-1": 'neutral-1',
+  // "graph-2": 'accent-2',
+  // "graph-3": 'neutral-2',
```

Comment:
we'll need to add these on the base.d.ts file

Reply:
Yep